open_event/views/admin/models_views/session.py

```diff
@@ -0,0 +1,17 @@
+from flask_admin import expose
+from flask_admin.contrib.sqla import ModelView
+from flask import request, url_for, redirect
+from ....helpers.data import DataManager
+from ....helpers.data_getter import DataGetter
+
+
+class SessionView(ModelView):
+
+    @expose('/new/<user_id>/<hash>/', methods=('GET', 'POST'))
+    def create_view(self, event_id, user_id, hash):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'hash'](https://www.codacy.com/app/triplez/open-event-orga-server/file/2674060820/issues/source?bid=3233864&fileBranchId=3324174#l11)